src/algorithm/config.jl

```diff
@@ -2,6 +2,12 @@ abstract type AbstractBranchingRule end
 abstract type AbstractVariableBranchingRule <: AbstractBranchingRule end
 struct MostInfeasible <: AbstractVariableBranchingRule end
 struct PseudocostBranching <: AbstractVariableBranchingRule end
+mutable struct StrongBranching <: AbstractVariableBranchingRule
+    μ::Real
+    strong_branching_model::Gurobi.Optimizer
```

Comment:
Think you can just remove this field entirely, let's just build a new model from scratch each time we do strong branching.

Reply:
(We can revisit this decision later if it's a bottleneck)